conda/libmesh-vtk/meta.yaml

```diff
@@ -20,32 +20,30 @@ build:
 
 requirements:
   build:
-    - {{ compiler('cxx') }}
-    - {{ moose_cxx }}
     - {{ moose_mpich }}
     - cmake
     - ninja
     - {{ moose_libglu }}                # [linux]
     - {{ moose_mesalib }}               # [linux]
-    - {{ cdt('libxt-devel') }}          # [linux]
-    - {{ cdt('libsm-devel') }}          # [linux]
-    - {{ cdt('libx11-devel') }}         # [linux]
-    - {{ cdt('libice-devel') }}         # [linux]
-    - {{ cdt('libxext-devel') }}        # [linux]
-    - {{ cdt('mesa-libgl-devel') }}     # [linux]
-    - {{ cdt('xorg-x11-proto-devel') }} # [linux]
+    - libxt-devel-cos7-x86_64           # [linux]
+    - libsm-devel-cos7-x86_64           # [linux]
+    - libx11-devel-cos7-x86_64          # [linux]
+    - libice-devel-cos7-x86_64          # [linux]
+    - libxext-devel-cos7-x86_64         # [linux]
+    - mesa-libgl-devel-cos7-x86_64      # [linux]
+    - xorg-x11-proto-devel-cos7-x86_64  # [linux]
 
   run:
     - {{ moose_mpich }}
     - {{ moose_libglu }}                # [linux]
     - {{ moose_mesalib }}               # [linux]
-    - {{ cdt('libxt-devel') }}          # [linux]
-    - {{ cdt('libsm-devel') }}          # [linux]
-    - {{ cdt('libx11-devel') }}         # [linux]
-    - {{ cdt('libice-devel') }}         # [linux]
-    - {{ cdt('libxext-devel') }}        # [linux]
-    - {{ cdt('mesa-libgl-devel') }}     # [linux]
-    - {{ cdt('xorg-x11-proto-devel') }} # [linux]
+    - libxt-devel-cos7-x86_64           # [linux]
+    - libsm-devel-cos7-x86_64           # [linux]
+    - libx11-devel-cos7-x86_64          # [linux]
+    - libice-devel-cos7-x86_64          # [linux]
+    - libxext-devel-cos7-x86_64         # [linux]
+    - mesa-libgl-devel-cos7-x86_64      # [linux]
+    - xorg-x11-proto-devel-cos7-x86_64  # [linux]
```

Comment:
You don't suppose these being updated with `mamba update --all` will break an installation of moose-libmesh-vtk? We may need to pin these like we do with {{ moose_mpich }}. Not sure though. Maybe allowing these libraries to update (without a new build of moose-libmesh-vtk) is fine...

Reply:
Pinning `moose_sysroot` via `moose-mpich` as we do should pin them (one of the versions here is 2018, so I don't think they'll be updated again). But would you like me to make the change? I think this fix will be temporary as conda-forge moves to a CentOS 7 default late this year into next.

Reply:
Right, that's how it normally works. These CDT packages are meant to be very stable, but I also see sense in making things a bit more foolproof. I'll add it in the next round of commits. :D

Reply:
lol, well this may all be moot, cause perhaps we don't need to be so bound to a specific version for VTK runtime?

Reply:
Since this CDT bit of conda is entirely new to me, I don't have a solid intuition on it. Perhaps better to be a bit conservative here since we know the current versions work. I would like to return to the more general syntax when we move to miniforge and conda-forge updates their minimums.